addons/payment_authorize/controllers/main.py

```diff
@@ -37,6 +37,32 @@ def authorize_s2s_create_json(self, **kwargs):
         acquirer = request.env['payment.acquirer'].browse(acquirer_id)
         return acquirer.s2s_process(kwargs)
 
+    @http.route(['/payment/authorize/s2s/create_json_3ds'], type='json', auth='public', csrf=False)
+    def authorize_s2s_create_json_3ds(self, **kwargs):
+        new_id = request.env['payment.acquirer'].browse(int(kwargs.get('acquirer_id'))).s2s_process(kwargs)
+
+        if not new_id:
+            res = {
+                'result': False,
+            }
+            return res
+
+        token = request.env['payment.token'].browse(new_id)
+
+        res = {
+            'result': True,
+            'id': new_id,
+            'short_name': token.short_name,
+            '3d_secure': False,
+            'verified': False,
+        }
+
+        if kwargs.get('verify_validity'):
```

Comment:
Among the `kwargs`, `acquirer_id` and `verify_validity` seem to be relevant for multiple acquirers, so it would be more elegant to make them explicit parameters of the route methods, possibly with default values.